src/components/multi_select.js

```diff
@@ -104,8 +106,10 @@ export class MultiSelect extends PureComponent {
       searchValue,
       withGrouping,
       generateClassName,
-      listRenderer
+      listRenderer,
+      lockedItems
```

Comment:
as we talked, please remove this property, disabled is passed as a field in items

Reply:
@MaksimDrobchak as this changes the approach fully, please close this PR and create new one